pkg/services/ngalert/notifier/file_store.go

```diff
@@ -62,6 +63,46 @@ func (fileStore *FileStore) FilepathFor(ctx context.Context, filename string) (s
 	return fileStore.pathFor(filename), err
 }
 
+// GetFullState returns a base64-encoded string containing the Alertmanager's silences and nflog.
+func (fileStore *FileStore) GetFullState(ctx context.Context) (string, error) {
```

Comment:
```suggestion
func (fileStore *FileStore) GetFullState(ctx context.Context, filenames ...string) (string, error) {
```

`file_store.go` has no notion of which keys to get from the kv store so let's make sure we don't break the abstraction by hardcoding keys you don't know about.

Once you've done that you can use the keys by passing them at the callsite, that we have in the notifier package if you make them public `notificationLogFilename` and `silencesFilename`